src/encoded/visualization.py

```diff
@@ -300,6 +315,85 @@ def format_bucket_result(bucket_result, returned_buckets, curr_field_depth = 0):
     return ret_result
 
 
+@view_config(route_name='recently_released_datasets', request_method=['GET'])
+@debug_log
+def recently_released_datasets(context, request):
+    
+    MAX_BUCKET_COUNT = 6 # default max rows
+    
+    size = request.params.get("max_row_count", MAX_BUCKET_COUNT)
+
+    # default parameters
+    search_param_lists = deepcopy(DEFAULT_BROWSE_PARAM_LISTS)
+    # append any other search parameters
+    for k,v in request.params.items():
+        if k == 'sort' or k == 'max_row_count':
+            continue
+        if k not in search_param_lists:
+            search_param_lists[k] = [v]
+        else:
+            search_param_lists[k].append(v)
+    #override
+    search_param_lists['status'] = ['released']
+    search_param_lists['dataset_label!'] = ['No value']
+
+    fields_to_aggregate_for = ['dataset_label']
+
+    primary_agg = {
+        "field_0" : {
+            "terms" : {
+                "field" : "embedded." + fields_to_aggregate_for[0] + ".raw",
+                # "missing" : TERM_NAME_FOR_NO_VALUE,
+                "order": { "public_release": "desc"},
+                "size" : size
+            },
+            "aggs" : deepcopy(RECENTLY_RELEASED_EXPSETS_AGGREGATION_DEFINITION)
+        }
+    }
+
+    primary_agg.update(deepcopy(RECENTLY_RELEASED_EXPSETS_AGGREGATION_DEFINITION))
```

Comment:
Use of `deepcopy` I'd say is unnecessary since you are not modifying the constant in any scenario